src/index.js

```diff
@@ -1,27 +1,38 @@
+// @flow
+
 /**
  * Multihash implementation in JavaScript.
  *
  * @module multihash
  */
 'use strict'
 
-const bs58 = require('bs58')
+import bs58 from 'bs58'
+import varint from 'varint'
+import {names, codes, defaultLengths, type Name, type Code} from "./constants"
```

Comment:
@Gozala @vmx given that this enables us to do dynamic imports, does this mean that we finally can also do treeshaking? (https://github.com/ipfs/js-ipfs/issues/951)